nixos/tests/sway.nix

```diff
@@ -45,6 +45,10 @@ import ./make-test-python.nix ({ pkgs, lib, ... }: {
           regular2 = foreground;
```

Comment:
We can remove this now, right?

Reply:
Yeah, the entire foot config I think (unless we want to return to ORC? Not sure how common font rendering issues are vs. how finicky ORC is (doesn't work reliably in interactive mode, etc.))